api/helpers/game-states/moves/seven-scuttle/execute.js

```diff
@@ -0,0 +1,58 @@
+const GamePhase = require('../../../../../utils/GamePhase.json');
+
+module.exports = {
+  friendlyName: 'Seven_Scuttle',
+
+  description: 'Returns new GameState resulting from requested scuttle move',
+
+  inputs: {
+    currentState: {
+      type: 'ref',
+      description: 'The latest GameState before the requesting player scuttles',
+      required: true,
+    },
+    /**
+     * @param { Object } requestedMove - Object describing the request to play scuttle
+     * @param { String } requestMove.cardId - Card Played 
+     * @param { String } requestMove.targetId - Card Targeted
+     * @param { MoveType.SEVEN_SCUTTLE } requestedMove.moveType - Specifies that this a scuttle
+     */
+    requestedMove: {
+      type: 'ref',
+      description: 'The move being requested. Specifies moveType, card played and targeted',
+    },
+    playedBy: {
+      type: 'number',
+      description: 'Player number of player requesting move.',
+    },
+  },
+  sync: true,
+  fn: ({ currentState, requestedMove, playedBy }, exits) => {
+    let result = _.cloneDeep(currentState);
+
+    const { cardId, targetId } = requestedMove;
+
+    const player = playedBy ? result.p1 : result.p0;
```

Comment:
We don't actually need to the player as a variable because the playedCard comes from the deck not their hand